### PR TITLE
replace pytoml with toml (with OrderedDict constructor)

### PR DESCRIPTION
**Pull request by illwieckz**
also add a `requirements.txt` file

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+argparse
+psutil
+toml>=0.9.0
```
Comment (necessarily-equal) [inline requirements.txt:3]: On my side, the packages needed seem to be:
* colorama
* psutil
* toml
* pillow

```
afontain@thinkpad-nixos ~/unv$ Urcheon/bin/urcheon                             
/usr/bin/env: ‘python3’: No such file or directory
afontain@thinkpad-nixos ~/unv$ nix-shell -p python38 --run Urcheon/bin/urcheon  Traceback (most recent call last):
  File "Urcheon/bin/urcheon", line 18, in <module>
    from Urcheon import Urcheon
  File "/home/antoine/unv/Urcheon/Urcheon/Urcheon.py", line 12, in <module>
    from Urcheon import Pak
  File "/home/antoine/unv/Urcheon/Urcheon/Pak.py", line 11, in <module>
    from Urcheon import Action
  File "/home/antoine/unv/Urcheon/Urcheon/Action.py", line 11, in <module>
    from Urcheon import FileSystem
  File "/home/antoine/unv/Urcheon/Urcheon/FileSystem.py", line 11, in <module>
    from Urcheon import Ui
  File "/home/antoine/unv/Urcheon/Urcheon/Ui.py", line 12, in <module>
    from colorama import Fore, Style, init
ModuleNotFoundError: No module named 'colorama'

[…]

afontain@thinkpad-nixos ~/unv$ nix-shell -p python38 python38Packages.{colorama,psutil,toml,pillow}  --run Urcheon/bin/urcheon
usage: urcheon [-h] <stage> [stage arguments]
```

* colorama comes from Ui.py
* PIL (pillow) comes from Bsp.py

File: Urcheon/Texset.py [hunk 2]
```diff
@@ -16,10 +16,12 @@
 from collections import OrderedDict
 import logging
 import os
-import pytoml
 import shutil
 import subprocess
 import tempfile
+import toml
+
+# FIXME: do we need OrderedDict toml constructor here?
```
Comment (necessarily-equal) [inline Urcheon/Texset.py:24]: I'll let you figure that out :X

File: requirements.txt (hunk 1 above)
Comment (illwieckz) [reply requirements.txt:3]: ah yes I forgot them

File: Urcheon/Texset.py (hunk 2 above)
Comment (illwieckz) [reply Urcheon/Texset.py:24]: I did some test, and default non-ordered constructor is not faster anyway.

**Pull request merged**